test/entity.dart

```diff
@@ -63,4 +64,68 @@ class TestEntity {
     omit = -1;
     disregard = 1;
   }
+
+  @Unique(type:OBXPropertyType.Byte)
```

Comment:
`@Unique` and `@Index` should be annotations in addition to `@Property`, no?

Reply:
Oh, wait. I thought in Dart `@Property` is always required, but no. Still, if defining a type or UID it should be done with `@Property`, no? And there should be no need to expose `indexUid` to the user.

Reply:
Could indexUid be part of a back end front end contract?

Reply:
I'd keep it the same way [Java does it, i.e. separate annotations](https://docs.objectbox.io/entity-annotations#property-indexes-with-index). The code's already updated in such a way so I'm resolving this conversation.